### PR TITLE
 Cleanup of unused compsets and grids for v2

### PR DESCRIPTION
**Pull request by wlin7**
This PR continues to clean up obsolete compsets from v1 and the associated
entries in the config files and the use_case files.
    
 Default settings in namelist_defaults_eam.xml for non-SE grids and older physpkg
 are also removed.
    
 Several new compset aliases are also added: WCYCL1950, WCYCL20TR, F1850 and F1950.
 F1850 uses the same forcing files as for WCYCL1850; F1950 and WCYCL1950 use the same 
 forcings as for A_WCYCL1950S_CMIP6_HR (instead of _LR) in v1 -- HR uses the 1950-1960 
 mean climatology while LR uses a single year of 1950).

 The longname for WCYCL1950 and WCYCL20TR (and pre-existing W1850) will need to be updated
 after PR #4085. So do hThe compset pattern matching for EAM use caae.
    
 Tests using the following compsets  will be renamed as 'v1' is now dropped from the compset names.
 F-EAMv1-AQP1, F-EAMv1-RCEMIP. There are five such tests.
    
[BFB]

### CODE REVIEW

**Comment by wlin7** [comment]
Hi All, please pay extra attention to namelist_defaults_eam.xml if ok to remove that many settings (eul and fv dycores and grids,  cam3, cam4, cam5 phys). T42 (64x128) is retained as it is used for an alternative configuration of SCM. Thanks.

File: components/eam/bld/namelist_files/use_cases/1950_eam_CMIP6.xml [hunk 1]
```diff
@@ -23,91 +23,6 @@
 <prescribed_volcaero_type    >CYCLICAL						       </prescribed_volcaero_type    >
 <prescribed_volcaero_cycle_yr>1                                                        </prescribed_volcaero_cycle_yr>
```
Comment (rljacob) [inline components/eam/bld/namelist_files/use_cases/1950_eam_CMIP6.xml:25]: Is the large block below removed because those are now defaults?

Comment (wlin7) [reply components/eam/bld/namelist_files/use_cases/1950_eam_CMIP6.xml:25]: Exactly, we move all physics parameters to the default file.

**Review by whannah1 — APPROVED** [review]
I perused through all the changes and didn't see any potential problems. This is a very refreshing cleanup, nice work @wlin7 !

**Review by jonbob — APPROVED** [review]
Looks good to me -- thanks for taking this on

**Review by singhbalwinder — APPROVED** [review]
Thanks @wlin7 ! Looks good to me!

**Comment by wlin7** [comment]
Thanks @golaz , @rljacob , @whannah1 , @jonbob , @bogensch , @singhbalwinder for the quick reviews to ensure all is ok. I did miss some in the initial commits which are now corrected before the new round of reviews from you all. Preliminary tests look good (A_WCYCL1850, wcprod, SCM, AQP1 and generic FC5AV1C-L). 

Note: This PR will be merged after #4085. Conflicts in config_compsets and config_components expected. lname for WCYCL1950 will be updated to reflect the new format as in #4085.

**Review by tangq — APPROVED** [review]
I looked at the changes. The clean up is really nice and I don't see any issues. Great job!

**Review by golaz — APPROVED** [review]

**Review by xuezhengllnl — APPROVED** [review]
Every change looks fine to me. The remaining compsets look good except It's not very clear to me what 2010S_EAM_CMIP6 and 20TRS_EAM*** are.
Thanks,
-Xue

**Comment by wlin7** [comment]
Thanks for the careful reviews, @tangq  and @xuezhengllnl . Indeed there are some other compsets that we may eventually want to remove as well. The compsets @xuezhengllnl mentioned above are either still in use (F2010SC5-CMIP6 in wcprod test) or as potential templates for specific future compsets (e.g., superfast and BGC related).

**Review by rljacob — APPROVED** [review]

**Comment by rljacob** [comment]
We can always change the test to match a new compset.

**Comment by wlin7** [comment]
Merged to next.

**Comment by wlin7** [comment]
Re-merged to next after 64846f9 for F-EAM-RCEMIP.

**Comment by wlin7** [comment]
Merged to master.

**Pull request merged**